src/vivarium_gates_mncnh/components/intervention.py

```diff
@@ -523,11 +514,6 @@ def __init__(self):
         self.risk_specific_shift_pipeline_name = (
             f"{self.risk.name}_on_{self.target.name}.risk_specific_shift"
         )
-        self.raw_gestational_age_exposure_column_name = "gestational_age_exposure"
```

Comment:
The rest of the changes in this file (except index handling) are to use the raw gestational age exposure pipeline instead of the state table column.